cluster/create_workflow.go

```diff
@@ -48,7 +56,33 @@ func CreateClusterWorkflow(ctx workflow.Context, input CreateClusterWorkflowInpu
 			ClusterID: input.ClusterID,
 		}
 
-		err := workflow.ExecuteActivity(ctx, DownloadK8sConfigActivityName, activityInput).Get(ctx, nil)
+		err := workflow.ExecuteActivity(ctx, DownloadK8sConfigActivityName, activityInput).Get(ctx, &configSecretID)
+		if err != nil {
+			return err
+		}
+	}
+
+	{
+		workflowInput := clustersetup.WorkflowInput{
+			ConfigSecretID: fmt.Sprintf("brn:%d:secret:%s", input.OrganizationID, configSecretID),
+			Cluster: clustersetup.Cluster{
+				ID:   input.ClusterID,
+				UID:  input.ClusterUID,
+				Name: input.ClusterName,
+			},
+			Organization: clustersetup.Organization{
+				ID:   input.OrganizationID,
+				Name: input.OrganizationName,
+			},
+		}
+
+		cwo := workflow.ChildWorkflowOptions{
+			ExecutionStartToCloseTimeout: 30 * time.Minute,
+			TaskStartToCloseTimeout:      40 * time.Minute,
+		}
+		ctx := workflow.WithChildOptions(ctx, cwo)
+
+		err := workflow.ExecuteChildWorkflow(ctx, clustersetup.WorkflowName, workflowInput).Get(ctx, nil)
```

Comment:
Could define `err` in the `if` to keep scope as tight as possible.

Reply:
I don't think it's necessary. The call is already scoped.